apps/condo/domains/ticket/tasks/sendTicketCommentCreatedNotifications.js

```diff
@@ -1,78 +1,130 @@
+        const lang = get(COUNTRIES, [organizationCountry, 'locale'], conf.DEFAULT_LOCALE)
+
+        setLocaleForKeystoneContext(context, lang)
+
+        const createdComment = await getById('TicketComment', commentId)
+        const commentAuthor = await getById('User', createdComment.user)
+        const commentAuthorType = get(commentAuthor, 'type', STAFF)
+        const commentType = get(createdComment, 'type', ORGANIZATION_COMMENT_TYPE)
+        const ticketStatus = await getById('TicketStatus', ticket.status)
+        const ticketUrl = `${conf.SERVER_URL}/ticket/${ticketId}`
+        const classifier = await TicketClassifier.getOne(context, { id: ticket.classifier })
+
+        const OpenTicketMessage = i18n(`notification.messages.${TICKET_COMMENT_CREATED_TYPE}.telegram.openTicket`, { locale: lang })
+        const TicketStatusName = i18n(`ticket.status.${ticketStatus.type}.name`, { locale: lang })
+        const TicketUnitType = i18n(`field.UnitType.prefix.${ticket.unitType}`, { locale: lang }).toLowerCase()
+        const CommentTypeMessage = getCommentTypeMessage(commentType, commentAuthorType, lang)
+
+        const ticketClassifier = buildFullClassifierName(classifier)
+
+        const users = await getUsersAvailableToReadTicketByPropertyScope({
+            ticketOrganizationId: ticket.organization,
+            ticketPropertyId: ticket.property,
+            ticketExecutorId: ticket.executor,
+            ticketAssigneeId: ticket.executor,
+            ticketCategoryClassifierId: ticket.categoryClassifier,
+        })
+        const usersWithoutSender = users.filter(userId => userId !== createdComment.user)
+
+        for (const employeeUserId of usersWithoutSender) {
+            await sendMessage(context, {
+                lang,
+                to: { user: { id: employeeUserId } },
+                type: TICKET_COMMENT_CREATED_TYPE,
+                meta: {
+                    dv: 1,
+                    data: {
+                        organizationId: organization.id,
+                        organizationName: organization.name,
+                        commentId,
+                        commentContent: createdComment.content || EMPTY_CONTENT,
+                        commentType: createdComment.type,
+                        CommentTypeMessage,
+                        commentCreatedAt: dayjs(createdComment.createdAt).format('YYYY-MM-DD HH:mm'),
+                        ticketId,
+                        ticketDetails: ticket.details,
+                        ticketClassifier,
+                        ticketNumber: ticket.number,
+                        ticketStatus: TicketStatusName,
+                        ticketAddress: ticket.propertyAddress,
+                        ticketUnit: ticket.unitName ? `${TicketUnitType} ${ticket.unitName}` : EMPTY_CONTENT,
+                        userId: employeeUserId,
+                        userName: commentAuthor.name,
+                        url: ticketUrl,
```

Comment:
It would be nice to see new features listed at `sendTicketCommentCreatedNotifications.spec.js`

Reply:
For example, you can check new structure of message

Reply:
done